doc/using-logging.rst

```diff
@@ -0,0 +1,117 @@
+=====================
+ Configuring Logging
+=====================
+
+One common use of ZConfig is to configure the Python :mod:`logging`
+framework. ZConfig provides one simple convenience function to do this:
+
+.. autofunction:: ZConfig.configureLoggers
+
+
+This function takes the configuration text::
+
+    >>> from ZConfig import configureLoggers
+    >>> configureLoggers('''
```

Comment:
No, I'm suggesting moving the configuration text into a separate block:

    For example, consider the logging configuration in the file ``logging.conf``:

       .. code-block:: xml
          ...

Then, in your programming example, show loading the configuration from the file.  That too would benefit from being freed from doctest clutter. :) Not because there's a lot of Python code, but because showing an interactive interpreter doesn't add anything, especially since there's no output. 

OTOH, the example below that shows the effect of the logging config *does* benefit from the doctest treatment.